projects/plugins/jetpack/class.jetpack.php

```diff
@@ -13,6 +13,7 @@
 use Automattic\Jetpack\Connection\Webhooks as Connection_Webhooks;
```

Comment:
sync_idc_optin and get_sync_error_idc_optin need to call the IDC packages version.

Reply:
mark these functions as deprecated.
Update all references in Jetpack to call IDC versions.